userbenchmark/optim/run.py

```diff
@@ -405,33 +406,48 @@ def run_model(modelName, device, Optim, defaults, maybe_pt2_):
 
         print(f'{datetime.datetime.now()}     python -m userbenchmark.optim.run -m {modelName} -d {device}' +
               f' -o {Optim.__name__} --df "{defaults_to_str(defaults)}" -f {maybe_pt2_}')
+
+        compile_r = None
+        sub_label = f'{modelName}, {optim.__class__.__name__}, {device}'
+        description = pt2_description + defaults_to_str(defaults)
+
+        # Get compile time by running 5 times and subtracting
+        #   first entry - avg(entries 3 through 5)
+        # skipping the second entry due to high variance of first cache hit
```

Comment:
Huh, I'm not sure why this is. The second run has higher variance than later runs?

Reply:
We also might want to do more runs, I think we do five warmup runs before taking a measurement common.py. 3 runs are needed to initialize cudagraphs. So that could possibly be it.

Reply:
The 5 warmup runs are exactly what PT2 bms do today + the compile time is based on those 5 warmup runs. I'll do 10 runs though...would you agree that the first run (which is significantly longer in all cases i've seen) should be what's taken as the first operand?

aka...would you agree compile time is close to:
```
first_run - avg(the last n runs)
```
If I run it 10 times, I will make n = 5.

Reply:
Like today, we basically already do 5 warmup runs before calculating runtime. This calculation is purely for compiletime

Reply:
cool sounds good, yeah this appears to be right. Thought avg(last n runs) was used for the runtime estiamte.